include/networkit/algebraic/MatrixTools.hpp

```diff
@@ -11,7 +11,7 @@
 #include <atomic>
 #include <cmath>
 
-#include <networkit/algebraic/Globals.hpp>
+#include <networkit/algebraic/AlgebraicGlobals.hpp>
```

Comment:
Hm? Why is this changed? (Instead of doing this in the first commit?)

Reply:
Ah sorry, instead of using `git commit --fixup` to fix the first commit I used `git commit --amend`, I'll fix this now.